components/cluster/service-hacluster/service-hacluster.p5m

```diff
@@ -36,9 +36,7 @@ set name=pkg.description value="$(COMPONENT_SUMMARY)"
 
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
-group groupname=haclient gid=66
```

Comment:
Why do we avoid shipping user/group hacluster?

Reply:
This goes to hacluster-common as in other packages.
https://github.com/grueni/oi-userland/blob/cf6f24bb43bc3e22ad9a9325ccd1e01936a40a93/components/cluster/hacluster-common/hacluster-common.p5m